lib/ansible/modules/windows/win_iis_webapppool.ps1

```diff
@@ -1,24 +1,11 @@
 #!powershell
-
-# (c) 2015, Henrik Wallström <henrik@wallstroms.nu>
-#
 # This file is part of Ansible
-#
-# Ansible is free software: you can redistribute it and/or modify
-# it under the terms of the GNU General Public License as published by
-# the Free Software Foundation, either version 3 of the License, or
-# (at your option) any later version.
-#
-# Ansible is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU General Public License for more details.
-#
-# You should have received a copy of the GNU General Public License
-# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
 
-# WANT_JSON
-# POWERSHELL_COMMON
+# Copyright: (c) 2015, Jon Hawkesworth (@jhawkesworth) <figs@unity.demon.co.uk>
```

Comment:
seem to have lost attribution for Henrik - I merely did some fixing, but Henrik is original module author:

-# (c) 2015, Henrik Wallström <henrik@wallstroms.nu>

Reply:
My bad, will add it back in.

Reply:
Thanks for the pickup, should be fixed now.